python/pyspark/ml/torch/distributor.py

```diff
@@ -330,6 +340,7 @@ def __init__(
         num_processes: int = 1,
         local_mode: bool = True,
         use_gpu: bool = True,
+        spark: Optional[SparkSession] = None,
```

Comment:
Do we need this ? Why not using active session ?

Reply:
right now, just for test purpose, I want to specify the session in test for debug